src/Conf.cpp

```diff
@@ -358,6 +381,83 @@ void Conf::setDataCenter(const ConfParser::Node* node)
     }
 }
 
+void Conf::setCustomCommand(const ConfParser::Node* node)
+{
+    if (!node->sub) {
+        Throw(InvalidValue, "%s:%d CustomCommand require scope value", node->file, node->line);
+    }
+    for (auto p = node->sub; p; p = p->next) {
+        mCustomCommands.push_back(CustomCommandConf{});
+        auto& cc = mCustomCommands.back();
+        CustomCommandConf::init(cc, p->key.c_str(), Command::Sentinel);
+        auto s = p->sub;
+        for (;s ; s = s->next) {
+            if (setInt(cc.minArgs, "MinArgs", s, 2)) {
+            } else if (setInt(cc.maxArgs, "MaxArgs", s, 2, 9999)) {
+            } else if (setCommandMode(cc.mode, "Mode", s)) {
+            } else {
+                Throw(UnknownKey, "%s:%d unknown key %s", s->file, s->line, s->key.c_str());
+            }
+        }
+        if (cc.maxArgs < cc.minArgs) {
+           Throw(InvalidValue, "%s:%d must be MaxArgs >= MinArgs", p->file, p->line);
+        }
+        Command cmd;
+        CustomCommandConf::convert(cc, cmd);
+        Command::addCustomCommand(cmd);
+    }
```

Comment:
like below, to avoid mCustomCommands changed after addCustomCommand
```
        //Command cmd;
        //CustomCommandConf::convert(cc, cmd);
        //Command::addCustomCommand(cmd);
    }
    for (auto& cc : mCustomCommands) {
        Command::addCustomCommand(cc);
    }
```